### DISCUSSION TITLE
Getting latex to work (newbie)

### DISCUSSION THREAD

**Discussion by mauricebis** [xournalpp/xournalpp#3545] [Q&A] [answered]
I installed the app through the stable appimage (Ubuntu 18.04). I've latex installed on my machine but when I want to insert a Latex formula in xournalapp, the OK button is greyed. Also, in settings, I don't understand what is meant by regular Latex template. I just want to insert occasionally latex maths formula on the page. Thanks.

**Comment by rolandlo** [comment, marked as answer]
@mauricebis Probably you need to change `default_template.tex` to `legacy_template.tex` in the LaTeX panel of menu `Edit > Preferences`. Ubuntu 18.04 doesn't have the `scontents` package and will therefore fail most likely.

**Reply by mauricebis** [reply]
Yes. That dit it. Many thanks.
